src/libraries/System.Net.Quic/src/System/Net/Quic/Interop/msquic.cs

```diff
@@ -108,72 +110,19 @@ public NativeTypeNameAttribute(string name)
         public string Name => _name;
     }
 
-    [StructLayout(LayoutKind.Explicit)]
-    internal struct QuicAddrFamilyAndLen
-    {
-        [FieldOffset(0)]
-        public ushort sin_family;
-        [FieldOffset(0)]
-        public byte sin_len;
-        [FieldOffset(1)]
-        public byte sin_family_bsd;
-    }
-
-    [StructLayout(LayoutKind.Sequential)]
-    internal unsafe struct QuicAddrIn
-    {
-        public QuicAddrFamilyAndLen sin_family;
-        public ushort sin_port;
-        public fixed byte sin_addr[4];
-    }
-
-    [StructLayout(LayoutKind.Sequential)]
-    internal unsafe struct QuicAddrIn6
-    {
-        public QuicAddrFamilyAndLen sin6_family;
-        public ushort sin6_port;
-        public uint sin6_flowinfo;
-        public fixed byte sin6_addr[16];
-        public uint sin6_scope_id;
-    }
-
-    [StructLayout(LayoutKind.Explicit)]
     internal struct QuicAddr
```

Comment:
Readers could (incorrectly) assume that the code was written by the MsQuic team when in fact we did, and since the QuicAddr struct does not seem to add anything useful over `SockAddr`, I would prefer removing it and using `SockAddr` directly instead.